external-crates/move/crates/move-analyzer/editors/code/README.md

```diff
@@ -21,13 +21,18 @@ definition, type on hover). Please see [Troubleshooting](#troubleshooting) for s
 the pre-built move-analyzer binary is not available for your platform or if you want to use move-analyzer
 binary stored in a different location.
 
-If you want to build and test Move code using the extension, you must install the `sui` binary on
+If you want to build, test, and trace Move code using the extension, you must install the `sui` binary on
 your machine - see [here](https://docs.sui.io/guides/developer/getting-started/sui-install) for
 instructions. The extension assumes that the `sui` binary is in your system path, but you can set
 its custom location location using VSCode's settings (`⌘` + `,` on macOS, or use the menu item *Code >
 Preferences > Settings*). Search for the `move.sui.path` user setting, set it to the new location of
 the `sui` binary, and restart VSCode.
 
+In order to trace Move code execution, the `sui` binary must be built with the `tracing` feature flag.
+If your version of the `sui` binary was not built with this feature flag, an attempt to trace test
+execution will fail. In this case you may have to build the `sui` binary from source following these
+[instructions](https://docs.sui.io/guides/developer/getting-started/sui-install#install-sui-binaries-from-source).
```

Comment:
Nit, consider:

```suggestion
If your version of the `sui` binary was not built with this feature flag, an attempt to trace test
execution will fail. In this case you can either install the `sui` binary using homebrew of chocolatey using these [instructions](LINK FILL ME IN), or build the `sui` binary from source following these
[instructions](https://docs.sui.io/guides/developer/getting-started/sui-install#install-sui-binaries-from-source).
```

Reply:
This would be the releant link: https://docs.sui.io/guides/developer/getting-started/sui-install#install-homebrew

Reply:
I was avoiding linking directly to brew/chocolatey install instructions as I don't know if these distros are already available with the feature flag enabled. If someone can confirm that indeed they are, then we should definitely do this